application/src/main/java/org/opentripplanner/routing/api/request/RouteRequest.java

```diff
@@ -188,10 +188,10 @@ public SortOrder itinerariesSortOrder() {
    * the cursor[if exist], not the date-time.
    * <p>
    * The direct mode is also unset when there is a page cursor because for anything other than the
-   * initial page we don't want to see direct results.
-   * <p>
-   * See also {@link org.opentripplanner.routing.algorithm.raptoradapter.router.FilterTransitWhenDirectModeIsEmpty},
-   * it uses a direct search to prune transit.
+   * initial page we don't want to see direct results. If the direct mode was given in the first request,
+   * the results of the direct mode search from the initial request are stored in the page cursor for use with
```

Comment:
I think you can be more precise on what is stored as `results` is a bit vague.

Reply:
I changed this comment